Makefile

```diff
@@ -0,0 +1,20 @@
+MOCHA := "mocha"
+NYC := "nyc"
+
+TESTS = test/*.js
+TESTS_WITH_TIZEN = test/*.js libs/brackets-server/embedded-ext/project/test/*.js
```

Comment:
How about using "${TESTS}" instead of the "test/*.js"?

Reply:
I will remove "test/*.js" per below since extension test lists two tests as below.
$(TESTS) $(TESTS_WITH_TIZEN)